setup-cf-kube-dist.sh

```diff
@@ -1,9 +1,19 @@
 #!/bin/bash
 
-target="$1"
+set -o nounset
+set -o errexit
 
-# EV 'PIPELINE_PREFIX' = prefix to pipeline name for local
-#                        customization of test pipelines
+# EV 'PIPELINE_PREFIX' = prefix to pipeline name for local customization of test
+#           pipelines
+
+PIPELINE_NAME=cf-kube-dist
+
+if test -z "${1:-}"; then
+    printf "Usage:\n%s <target>\n" "${0}" >&2
+    exit 1
+fi
+
+target="${1}"
```

Comment:
May want to try `${1:-}` and do some error messaging if it's unset, rather than whatever confusing default error we get.